jupyter/intel/ml/ubi9-python-3.11/utils/addons/apply.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# See https://github.com/jupyterlab/jupyterlab/issues/5463
+# This is a hack to apply partial HTML code to JupyterLab's `index.html` file
+# Look for the other duplicates in case a change is needed to this file
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+pf_url="https://unpkg.com/@patternfly/patternfly@6.0.0/patternfly.min.css"
```

Comment:
@jstourac was scared for a moment what would happen on disconnected cluster here! but the good news is that this stylesheet is fetched on users browser, and that does have general network access, or does it, actually?

it would work in the DevTestOps disconnected setup, but I imagine that companies that use disconnected setups for their clusters may also restrict the user's computers net access?

Reply:
This URL is downloaded as part of the script that runs during image build time. The CSS file becomes available locally for the page to use at runtime.